loader/src/main/scala/com/actian/spark_vector/loader/parsers/Parser.scala

```diff
@@ -68,10 +68,12 @@ sealed case class ArgOption[T: Read, O](
  *         Vector host name
  *   -vi <value> | --vectorInstance <value>
  *         Vector instance
+ *   -vo <value> | --vectorInstOffset <value>
```

Comment:
How about having a section of common (shared) options and then sections for specific options?